### ISSUE TITLE
Unable to type in birthday - only pickup possible ...

### ISSUE THREAD

**Issue by kangaroo72** [nextcloud/contacts#1473]
App-Version: 3.8.1 / NC 18.0.1

Dear devs,

I think there's already an issue, but I cannot find it.
When I want to add a birthday to a contact, I only can select it with the date-picker.
I'm unable to type in the date directly. This would be much faster.
Can you help there??

Thanks, Tom

**Comment by askuri** [comment]
I can confirm the problem on NC 17.0.3 with Contacts 3.1.8. All fields that have a date involved are affected.

**To Reproduce**
1. Open any contact that might or might not have a birthday field
2. Add a birthday field if there isn't one already
3. Change the date in the text field

**Expected behavior**
Date picker closes and value from the text field is taken.

**Actual behavior**
Date picker closes but value is reset to what was there before.

### Server configuration
**Operating system**: Ubuntu 18.04

**Web server:** Apache

**Database:** MySQL 5.7.29 

**PHP version:** 7.2.24

**Nextcloud version:** 17.0.3

**Contacts version:** 3.1.8

**Updated from an older Nextcloud or fresh install:** upgraded from older version

**Signing status:**
```
No errors have been found.
```

**List of activated apps:**
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**
```
Enabled:
  - accessibility: 1.3.0
  - activity: 2.10.1
  - calendar: 2.0.1
  - cloud_federation_api: 1.0.0
  - comments: 1.7.0
  - contacts: 3.1.8
  - dav: 1.13.0
  - federatedfilesharing: 1.7.0
  - federation: 1.7.0
  - files: 1.12.0
  - files_pdfviewer: 1.6.0
  - files_rightclick: 0.15.1
  - files_sharing: 1.9.0
  - files_trashbin: 1.7.0
  - files_versions: 1.10.0
  - files_videoplayer: 1.6.0
  - firstrunwizard: 2.6.0
  - gallery: 18.4.0
  - issuetemplate: 0.6.0
  - logreader: 2.2.0
  - lookup_server_connector: 1.5.0
  - nextcloud_announcements: 1.6.0
  - notes: 3.1.5
  - notifications: 2.5.0
  - oauth2: 1.5.0
  - password_policy: 1.7.0
  - privacy: 1.1.0
  - provisioning_api: 1.7.0
  - ransomware_protection: 1.5.1
  - recommendations: 0.5.0
  - serverinfo: 1.7.0
  - sharebymail: 1.7.0
  - support: 1.0.1
  - survey_client: 1.5.0
  - systemtags: 1.7.0
  - text: 1.1.1
  - theming: 1.8.0
  - twofactor_backupcodes: 1.6.0
  - updatenotification: 1.7.0
  - viewer: 1.2.0
  - workflowengine: 1.7.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_ldap
```

### Client configuration
**Browser:** Firefox 73.0

**Operating system:** Kubuntu 19.10

**CardDAV-clients:** DAVx5

### Logs
#### *Web server error log*
Not sure if this is relevant, but I get my webserver's error.log spammed with these messages: 
```
cat: /etc/ntp.conf: No such file or directory
```

### Nextcloud log
#### *data/nextcloud.log*
No relevant errors

#### Browser log
a) Javascript console: no messages come up when entering a date into the textfield and pressing enter
b) Network: no network activity when when entering a date into the text field and pressing enter.

**Comment by call-me-matt** [comment]
Remark: partial update - https://github.com/mengxiong10/vue2-datepicker/issues/290

**Comment by daiwai** [comment]
> Remark: partial update - [mengxiong10/vue2-datepicker#290](https://github.com/mengxiong10/vue2-datepicker/issues/290)
> But I am confused as it has been marked closed in [nextcloud/nextcloud-vue#137](https://github.com/nextcloud/nextcloud-vue/issues/137) ?

Are those really the same issues as this one? If I understood correctly, vue2-datepicker#290 is about changing a part of the date by mouse in the calendar UI, whereas this issue is about accepting inputs into the input element to which the calendar is attached.

**Comment by jangop** [comment]
Is this the same issue as #1865?
